rocketpy/utilities.py

```diff
@@ -31,5 +38,159 @@ def compute_CdS_from_drop_test(
 
     """
 
-    CdS = 2 * rocket_mass * gravity / ((terminal_velocity**2) * air_density)
-    return CdS
+    return 2 * rocket_mass * g / ((terminal_velocity**2) * air_density)
+
+
+# TODO: Needs tests
+
+
+def calculateEquilibriumAltitude(
+    rocket_mass,
+    CdS,
+    z0,
+    v0=0,
+    env=None,
+    eps=1e-3,
+    max_step=0.1,
+    seeGraphs=True,
+    g=9.80665,
+    estimated_final_time=10,
+):
+    """Returns a dictionary containing the time, altitude and velocity of the
+    system rocket-parachute in which the terminal velocity is reached.
+
+
+    Parameters
+    ----------
+    rocket_mass : float
+        Rocket's mass in kg.
+    CdS : float
+        Number equal to drag coefficient times reference area for parachute.
+    z0 : float
+        Initial altitude of the rocket in meters.
+    v0 : float, optional
+        Rocket's initial speed in m/s. Must be negative
+    env : Environment, optional
+        Environmental conditions at the time of the launch.
+    eps : float, optional
+        acceptable error in meters.
+    max_step: float, optional
+        maximum allowed time step size to solve the integration
+    seeGraphs : boolean, optional
+        True if you want to see time vs altitude and time vs speed graphs,
+        False otherwise.
+    g : float, optional
+        Gravitational acceleration experienced by the rocket and parachute during
+        descent in m/s^2. Default value is the standard gravity, 9.80665.
+    estimated_final_time: float, optional
+        Estimative of how much time (in seconds) will spend until vertical terminal velocity is reached. Must be positive. Default is 10. It can affect the final result if the value is not high enough. Increase the estimative in case the final solution is not founded.

```

Comment:
```suggestion
        Estimative of how much time (in seconds) will spend until vertical terminal 
        velocity is reached. Must be positive. Default is 10. It can affect the final 
        result if the value is not high enough. Increase the estimative in case the 
        final solution is not founded.

```